test/framework/src/main/java/org/elasticsearch/search/aggregations/metrics/AbstractGeoTestCase.java

```diff
@@ -263,6 +250,16 @@ public void setupSuiteScopeCluster() throws Exception {
         assertThat(totalHits, equalTo(2000L));
     }
 
+    private SpatialPoint computeCentroid(SpatialPoint[] points) {
```

Comment:
Makes sense. I think I have just not spent enough time thinking about compensated sums, so I was over optimistic about my simpler calculation in the test.